db/migrate/20250527151609_change_base_interface_type.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeBaseInterfaceType < ActiveRecord::Migration[7.0]
+  def up
+    Nic::Base.where(type: nil).update_all(type: 'Nic::Managed')
```

Comment:
I guess we need to convert `Nic::Base` or other possible types which are not in `allowed` ones?